addons/l10n_ma/models/res_company.py

```diff
@@ -0,0 +1,25 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from markupsafe import Markup
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_ma_ice = fields.Char("ICE", help='Common Company Identifier')
```

Comment:
I asked the PO on the task if it is different from `company_registry`, to avoid adding a field (and therefore allow doing the change in 17.0 instead of only master)